backend/database/db.go

```diff
@@ -26,30 +30,112 @@ func Init() *gorm.DB {
 
 	DB, err = gorm.Open(dialector, &gorm.Config{})
 
-	// Migration/Creation of data tables for DB
-	// NOTE: Order of table creation matters
-	// Dependent tables must be created in the right
-	// order so that relations and keys exist.
-	if !DB.Migrator().HasTable(&models.User{}) {
-		DB.Migrator().CreateTable(&models.User{})
-	}
-	if !DB.Migrator().HasTable(&models.Session{}) {
-		DB.Migrator().CreateTable(&models.Session{})
-	}
-	if !DB.Migrator().HasTable(&models.Journey{}) {
-		DB.Migrator().CreateTable(&models.Journey{})
-	}
-	if !DB.Migrator().HasTable(&models.Event{}) {
-		DB.Migrator().CreateTable(&models.Event{})
+	// Grab command line arguments
+	// --migrate-db 			- Perform the DB migration
+	// --seed-db=some-file.json	- Seed the DB with a JSON file
+	doMigration := flag.Bool("migrate-db", false, "Do DB migration")
```

Comment:
We should probably handle this with cobra in main.go or something.

Reply:
Going to stray away from Cobra and maybe break these out into separate files for specifically only running the migration or seeding.